modules/generative-databricks/clients/databricks.go

```diff
@@ -279,6 +282,10 @@ func (v *databricks) getApiKeyFromContext(ctx context.Context, apiKey, envVarVal
 	return "", fmt.Errorf("no api key found neither in request header: %s nor in environment variable under %s", apiKey, envVar)
 }
 
+func (v *databricks) getHeaderValue(ctx context.Context, header string) string {
```

Comment:
why add this function instead of using `modulecomponents.GetValueFromContext` directly?

Reply:
that's my style of writing :) can change it if you want :)

Reply:
As the function does nothing besides calling modulecomponents.GetValueFromContext I'd prefer if you'd remove `getHeaderValue`. We have too much copy&paste in the modules and the more is shared the better :)

Reply:
fixed!